static/js/modules/tables.js

```diff
@@ -605,14 +610,28 @@ DataTable.prototype.fetchSuccess = function(resp) {
 DataTable.prototype.fetchError = function() {
   var self = this;
 
-  setTimeout(function() {
-    $('.is-loading').removeClass('is-loading').addClass('is-unsuccessful');
-    self.$processing.hide();
-  }, helpers.LOADING_DELAY);
+  $('.filter__message').remove();
+
+  $('.filters .is-loading')
+    .removeClass('is-loading')
+    .addClass('is-unsuccessful')
+    .after($('<div class="filter__message filter__message--error">' +
+      '<strong>We had trouble processing your request</strong><br>' +
+      'Please try again. If you still have trouble ' +
+      '<span class="js-filter-feedback">let us know</a></div>'))
```

Comment:
I waffled between `a` and `button`, but seeing as how most of the triggers on the site are button, I went ahead and made the change. Thanks for the 👀